src/hoi4_world/countries/hoi4_country_converter.cpp

```diff
@@ -269,8 +269,11 @@ std::map<std::string, float> CalculateRawIdeologySupport(const std::vector<int>&
     const mappers::IdeologyMapper& ideology_mapper)
 {
    std::map<std::string, float> ideology_support;
+   std::map<std::string, std::map<std::string, float>> total_ideologies_per_law;
```

Comment:
Remove this when you're happy with how it's all working, or perhaps tie it to the overall debug setting.

Reply:
I"ll tie it to the overall debug setting. Since it'll be optimized away as long as it's not output, I'll keep the output in the ifdef, but keep the assignment where it is to reduce complexity.